repos/roslyn-tools.proj

```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommand>$(ProjectDirectory)\build$(ShellExtension) --pack --configuration $(Configuration)</BuildCommand>
+    <BuildArguments Condition="'$(OS)' == 'Windows_NT'">-pack -configuration $(Configuration)</BuildArguments>
+    <BuildArguments Condition="'$(OS)' != 'Windows_NT'">--pack --configuration $(Configuration)</BuildArguments>
```

Comment:
This is a minor case, but I suggest creating a prop for `-` vs. `--` to avoid duplication.